prow/config.yaml

```diff
@@ -310,9 +310,11 @@ postsubmits:
           - image: gcr.io/k8s-testimages/e2e-kubeadm:0.2
             args:
             - "--branch=$(PULL_REFS)"
-            - "--upload=gs://kubernetes-jenkins/logs"
-            - "--git-cache=/root/.cache/git"
             - "--clean"
+            - "--git-cache=/root/.cache/git"
+            - "--json"
+            - "--timeout=140"
```

Comment:
Should this match the value given in `ci-kubernetes-e2e-kubeadm-gce.env`? Which one overrides the other?